components/Main.js

```diff
@@ -56,7 +57,7 @@ export default function Main() {
         />
         <Stack.Screen
           name={routes.scan.name}
-          component={() => <Text>{routes.scan.name} placeholder</Text>}
+          component={ScanNewSecret}
```

Comment:
can we use the Screen suffix for all components which are screens?

Reply:
Yes, I'll create a `screens` directory inside components as well and move these components into there.